### ISSUE TITLE
Switch to grSim

### ISSUE THREAD

**Issue by justbuchanan** [RoboJackets/robocup-software#105]
Our simulator works pretty well, but the tuning isn't quite right and it's a little buggy.  There's an open source simulator called 'grSim' that's used by several teams and is pretty good.  Switching to it would make it easier to maintain our project and give us some better results.

**Comment by justbuchanan** [comment]
Reopening because it was brought up in the TC talk this year that the league may standardize on using grSim and move the project into the robocup-ssl github account.

**Comment by justbuchanan** [comment]
Is anyone interested in looking into this?  It would be nice for controls testing to model our 2015 bot in the simulator and it would be significantly easier to add it to grSim than to our current simulator.

Also see issue #246.

**Comment by jpfeltracco** [comment]
@jgkamat 
I've started working on this in the jeremy/grSim branch. I have kick on break beam implemented on the grSim side and I've spoofed ball sense so everything is minimally functional. We can drag robots around, move the ball, run some plays, and most of the simulator commands work.

- [ ] Get kick strength conversion correct
- [ ] Switch which side of the field yellow is initially placed on
- [ ] Add a RJ2015 model to grSim?
     - [ ] Tune whichever model we end up using
- [ ] Implement a "reset field" menu option
